app/ts/components/App.tsx

```diff
@@ -112,7 +106,7 @@ export function App() {
 		if (contentHashTimeoutRef.current !== null) clearTimeout(contentHashTimeoutRef.current)
 		contentHashTimeoutRef.current = window.setTimeout(() => {
 			contentHashTimeoutRef.current = null
-			if (!isValidContentHashString(contentHashInput.value)) return setError('Not valid content hash')
+			if (!isValidContentHashString(contentHashInput.value)) return setError('The content hash is not valid')
 			setError(undefined)
 			updateInfos(false)
 		}, 500)
```

Comment:
I wonder if this pattern can be extracted into a helper function that could be used instead of having to repeatedly apply the pattern throughout the code.

Reply:
its used once in the code. Or maybe I am misunderstanding you. The "is valid content hash, throw error", is used only here. `isValidContentHashString` is used more, but its already quite minimal function